iptables/iptables.go

```diff
@@ -26,3 +32,9 @@ func AddRule(appPort, metadataAddress, hostInterface, hostIP string) error {
 
 	return nil
 }
+
+// CheckInterfaceExists - validates the interface passed exists for the given system
+func CheckInterfaceExists(hostInterface string) error {
+	_, err := net.InterfaceByName(hostInterface)
```

Comment:
https://golang.org/pkg/net/#InterfaceByName does not support the iptables format.